docs/index.asciidoc

```diff
@@ -219,28 +214,22 @@ The host or ip to bind
 
 [id="plugins-{type}s-{plugin}-keystore"]
 ===== `keystore`
+deprecated[3.1.0, Use <<plugins-{type}s-{plugin}-ssl_certificate>> and <<plugins-{type}s-{plugin}-ssl_key>> instead]
 
   * Value type is <<path,path>>
   * There is no default value for this setting.
-  * This option is deprecated
 
 The JKS keystore to validate the client's certificates
 
-Note: This option is deprecated and it will be removed in the next major version of Logstash.
-Use `ssl_certificate` and `ssl_key` instead.
-
 [id="plugins-{type}s-{plugin}-keystore_password"]
 ===== `keystore_password`
+deprecated[3.1.0, Use <<plugins-{type}s-{plugin}-ssl_certificate>> and <<plugins-{type}s-{plugin}-ssl_key>> instead]
 
   * Value type is <<password,password>>
   * There is no default value for this setting.
-  * This option is deprecated
 
 Set the truststore password
 
-Note: This option is deprecated and it will be removed in the next major version of Logstash.
```

Comment:
Isn't this info emitted in the deprecation notice log? I think we can remove it from here (and for other deprecated options).